internal/pkg/cli/env_init.go

```diff
@@ -172,46 +189,137 @@ func (o *initEnvOpts) Execute() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateCustomizedResources() error {
+	var vpcImport, vpcConfig bool
+	if (o.ImportVPC.ID != "") || (o.ImportVPC.PublicSubnetIDs != nil) || (o.ImportVPC.PrivateSubnetIDs != nil) {
```

Comment:
you know I wonder if we should have `importVPCVars` and `adjustVPCVars` defined as separate structs above instead of anonymous.

This way we can have:
```go
func (v importVPCVars) IsSet() bool {
   if v.ID != "" {
       return true
   }
   if v.PublicSubnetIDs != nil {
       return true
   }
   if v.PrivateSubnetIDs != nil {
       return true
   }
   return false
}
```
This way most of this method becomes super simple

Reply:
this is super slick!